src/System.Runtime.Extensions/src/System/Environment.WinRT.cs

```diff
@@ -0,0 +1,124 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.Foundation.Metadata;
+using Windows.Storage;
+
+namespace System
+{
+    public static partial class Environment
+    {
+        private static string GetFolderPathCore(SpecialFolder folder, SpecialFolderOption option)
+        {
+            // For testing we'll fall back if the needed APIs aren't present
+            if (ApiInformation.IsTypePresent("Windows.Storage.UserDataPaths"))
+            {
+                return GetFolderPathCoreCurrent(folder, option);
+            }
+            else
+            {
+                return GetFolderPathCoreFallBack(folder, option);
+            }
+
+        }
+
+        private static string GetFolderPathCoreCurrent(SpecialFolder folder, SpecialFolderOption option)
```

Comment:
On Windows the API creates a very specific ACL for folders through the API. It is asking for trouble to just call CreateDirectory- I'm super hesitant to try laying down a folder with potentially compat-breaking ACLs.

That said, I'm digging into the behavior a bit more. I've just set up another clean install- I'll circle back with more details. (It may be moot as the folders that actually are exposed are all standard folders...)